### DISCUSSION TITLE
文章复制问题？

### DISCUSSION THREAD

**Discussion by foryouos** [theme-shoka-x/hexo-theme-shokaX#122] [Q&A] [answered]
请问文章无法通过Ctrl + C 或者鼠标右键复制，应该是我的问题，我看了那四个网站，出了第一个可以复制之外，其它的都不可以。使用的最新版本。可以复制，但是复制失败
今天使用复制一下写的代码，一直复制不过去，代码区域也没有复制选项，文字也无法复制，手机端也是同样问题。
![image](https://github.com/theme-shoka-x/hexo-theme-shokaX/assets/58133210/994ac75d-5859-488b-9ae9-3418df7461fa)

**Comment by zkz098** [comment]
疑似为新版本更新后的Bug，正在追踪具体出现版本

**Reply by zkz098** [reply]
0.2.6后开始出现此问题

**Reply by zkz098** [reply]
似乎是因为程序始终将nocopy视作true导致的，在nocopy下会删除代码块的复制并禁止转载
 但这并非预期行为

**Reply by zkz098** [reply]
![image](https://github.com/theme-shoka-x/hexo-theme-shokaX/assets/92242020/cbf45274-ba53-443a-8f80-f9702c22123e)
https://github.com/theme-shoka-x/hexo-theme-shokaX/commit/56fada5a57998e183824531cd3a21d6c8c00f8ce 导致的boolean均被视作string，且不可能为空，因此始终为真
正在研究修复方式

**Reply by zkz098** [reply, marked as answer]
0.2.10修复(计划发布)
